src/api/products/products.service.js

```diff
@@ -10,7 +13,37 @@ async function getById({ id }) {
   return product;
 }
 
+async function getRecommended({ userId }) {
+  const userOrders = await ordersService.getOrdersByUserId({ userId });
+  if (!(userOrders && userOrders.length >= 1)) {
+    // Retornamos los producto más recientes si no hay pedidos todavía
+    const products = await productsRepository.getAll();
+    return products;
```

Comment:
os gusta más devolver todos los productos como recomendados que devolver un error tipo: {msg: 'Si quieres ver productos recomendados realiza primero tu primera compra'} o algo así???

Reply:
a mí personalmente me gusta que se le recomiende las últimas novedades si no ha comprado todavía...tipo haciendo mas marketing, generándole necesidad de comprar. Me parece más atractivo que un mensaje pero es mi opinión, lo hacemos como queráis